source/vibe/web/rest.d

```diff
@@ -368,17 +349,15 @@ class RestInterfaceClient(I) : I
 	}
 
 	/// ditto
-	final @property void requestFilter(RequestFilter v) {
+	final @property void requestFilter(RequestFilter v)
+	{
 		m_requestFilter = v;
-		mixin (generateRestInterfaceSubInterfaceRequestFilter!I());
+		foreach (i, SI; Info.SubInterfaceTypes)
+			m_subInterfaces[i].requestFilter = v;
 	}
 
-	//pragma(msg, "subinterfaces:");
-	//pragma(msg, generateRestInterfaceSubInterfaces!(I)());
-	mixin (generateRestInterfaceSubInterfaces!I());
-
 	//pragma(msg, "restinterface:");
-	mixin RestClientMethods!I;
+	mixin(generateRestClientMethods!I());
```

Comment:
TBH I was hoping to get rid of string mixins in this module. What's the  rationale to bring them back here ?